packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-answer.tsx

```diff
@@ -30,6 +40,13 @@ export class AtomicSmartSnippetAnswer {
     this.emitCurrentHeight();
   }
 
+  public componentDidLoad() {
+    // Prevents initial transition
+    setTimeout(() => {
+      this.host.classList.add('animatable');
+    }, 0);
+  }
```

Comment:
I was able to see some piece of animation on load, setting the transition after the height is set does it. Open to other working solutions

Reply:
That's indeed pretty hackish 🤔 Can we use other lifecyle ? `componentDidRender` ?

Reply:
`componentDidLoad` is the last of them. I can investigate more on the smart snippet rendering loop though, @btaillon can surely help me with it

Reply:
Since from the perspective of this component there's no animation, I would change the class name from `animtable` to `loaded` or `rendered`.

I agree that `componentDidRender` may be preferable. If we use `componentDidRender`, then we should also remove the class on `componentWillRender`.